src/components/Whiteboard/ElementBehaviors/Resizable/ResizeHandle.tsx

```diff
@@ -124,30 +150,47 @@ export type DragEvent = {
 };
 
 export type ResizeHandleProps = {
-  handlePosition: ResizeHandlePosition;
-  containerWidth: number;
-  containerHeight: number;
   onDrag?: Dispatch<DragEvent>;
   onDragStart?: Dispatch<DragEvent>;
   onDragStop?: Dispatch<DragEvent>;
-};
+} & (LineElementProps | PolylineAndShapeElementsProps);
```

Comment:
I suggest to add a separate prop: `params` (or `resizeParams` or some better name) of the type `LineElementProps | PolylineAndShapeElementsProps`